MonoGame.Framework.Content.Pipeline/H264Importer.cs

```diff
@@ -16,5 +19,10 @@ public override VideoContent Import(string filename, ContentImporterContext cont
             var content = new VideoContent(filename);
             return content;
         }
+
+        public override VideoContent Import(Stream input, string virtualFilename, ContentImporterContext context)
+        {
+            throw new NotSupportedException("Video content must be stored on disk, not imported from a stream.");
```

Comment:
It seems I was incorrect in calling it incorrect, or it has changed since that time.  Only WAV content is loaded into memory.  Compressed audio just stores the file path.